fix_dir_perms/tasks/main.yml

```diff
@@ -0,0 +1,9 @@
+---
+
+- name: Fix directory permissions
+  file:
+    state: directory
+    path: "{{ fix_dir_perms_path }}"
+    owner: "{{ fix_dir_perms_owner }}"
+    group: "{{ fix_dir_perms_group }}"
+    mode: "{{ fix_dir_perms_mode }}"
```

Comment:
Just a reminder about `recursive` in case you forgot it

Reply:
@OmarIthawi didn't forget about `recurse`. In this case, the problem is that `gcsfuse` explicitly creates `/var/www/letsencrypt` which *implicitly* creates `/var/www`, setting its permissions/ownership as it does so. When that happens before nginx, it ends up being too restrictive. So the goal here is to surgically go in and fix the permissions on `/var/www` but we really don't want to touch anything further down than that (especially since the `letsencrypt` stuff is also mounted to `/etc/letsencrypt` so we need to be careful to not break permissions there).

Reply:
Great, thanks @thraxil